gateway-contracts/contracts/GatewayConfig.sol

```diff
@@ -22,7 +22,7 @@ contract GatewayConfig is IGatewayConfig, Ownable2StepUpgradeable, UUPSUpgradeab
     /// @dev they can still define their own private constants with the same name.
     string private constant CONTRACT_NAME = "GatewayConfig";
     uint256 private constant MAJOR_VERSION = 0;
-    uint256 private constant MINOR_VERSION = 1;
+    uint256 private constant MINOR_VERSION = 2;
```

Comment:
@RomanBredehoft, here's the minor version bump. But given [the ongoing thread](https://zama-ai.slack.com/archives/C088TGFV264/p1749649264576049), maybe we should wait for a decision to be made? 🤔